src/Stripe.net/Services/Sources/SourceCardOptions.cs

```diff
@@ -25,6 +26,9 @@ public class SourceCardOptions : INestedOptions
         [JsonProperty("currency")]
         public string Currency { get; set; }
 
+        [JsonProperty("default_for_currency")]
+        public bool? DefaultForCurrency { get; set; }
+
```

Comment:
Why is this here? `default_for_currency` is for *external accounts* so it doesn't match a class named `SourceCardOptions`

Reply:
Ah, added that as an override because it was showing as removed from something but I can't find which so will remove.

Reply:
Removed